docs/content/how-to/aws/troubleshooting.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Troubleshooting
+---
+
+# Troubleshooting
+
+## Disaster Recovery - Hosted Cluster Migration
+
+These are issues related with disaster recovery that we've identified and you could face during a Hosted Cluster migration.
+
+### The new workloads do not got scheduled in the new migrated cluster
+
+Everything looks normal, in the destination Management or Hosted Cluster and in the old Management and Hosted Cluster, but your new workloads do not schedule in your migrated Hosted Cluster (your old ones should work properly).
+
+Eventually your pods begin to fall down and the cluster status becomes degraded.
+
+1. First thing you need to check is the cluster operators and validate all of them work properly:
+```
+oc get co
+```
+
+2. If there are some of them degraded and with errors, please check the logs and validate things point to an OVN issue.
+```
+oc get co <Operator's Name> -o yaml
+```
+
+3. To solve the issue, we need to ensure the old Hosted Cluster is in pause ([we also need this](https://github.com/openshift/hypershift/pull/2265)) or deleted.
+
+4. Now we need to delete the OVN pods
+```
+oc --kubeconfig=${HC_KUBECONFIG} delete pod -n openshift-ovn-kubernetes --all
+```
+
+Eventually the Hosted Cluster will start self healing and the ClusterOperator will come back.
+
+**Cause:** The cause of this issue is after the Hosted Cluster Migration the KAS (Kube API Server) uses the same DNS name, but it points to different load balancer in AWS platform. Sometimes OVN does not behave correctly facing this situation.
+
+### The migration gets blocked in ETCD recovery
+
+The context around it's basically "I've edited the Hosted Cluster adding the `ETCDSnapshotURL` but the modification disappears and does not continue".
+
+The first simptom is the status of the Hypershift Operator pod, in this case the pod is usually in `CrashLoopBackOff` status.
+
+To solve this issue we need to:
+
+1. Kill the Hypershift Operator pod
+```
+oc delete pod -n hypershift -lapp=operator
+```
+
+2. Continue editing the pod, in order to add the `ETCDSnapshotURL` to the Hosted Cluster spec.
```

Comment:
Why would you edit the pod, shouldn't you edit the hosted cluster?

Reply:
yep, you're right, In my mind was also edit HostedCluster xD